data/human/navy missions intro.txt

```diff
@@ -0,0 +1,1496 @@
+			`	"Not quite." She replies brightly. "We've been experiencing some latency between Farpoint and Maelstrom. I'd need someone to fly through, then let me look at their logs so I can figure out what the trouble is."`
+			choice
+				`	"Sounds simple enough."`
+				`	"Sorry, I won't be able to help you either."`
+					decline
+			`	"Excellent." She smiles. "I'm Captain Jane Melgrew, by the way. Thank you for your help."`
+				accept
+
+	on accept
+		log "Volunteered to help track down the cause of some latency for the Republic."
+
+	on complete
+		payment 30000
+		dialog
+			`As you land on <destination>, Jane approaches you again. "Thank you on behalf of not only the Republic, but everyone who uses this network. I've transfered <payment> to your account."`
+
+
+
+mission "Republic Scouting 2"
+	landing
+	name "Cul-de-sac System Scouting"
+	description "Fly through the <waypoints> systems, then report to <destination> so the Navy can analyze your logs of pirate activity (if any)."
+	waypoint "Regulus"
+	waypoint "Fingol"
+	waypoint "Nocte"
+	waypoint "Ipsing"
+	source
+		government "Republic"
+	destination "Follower"
+	to offer
+		has "Republic Scouting 1: done"
+		not "chosen sides"
+		random < 40
+
+	on offer
+		dialog
+			`You receive a message from Jane, the Navy captain who asked you to fly through some uninhabited systems to the north. "I need some data from a few dead end systems," she says. "There might be some pirate fleets in the area but I just need logs again. Interested?"`
+
+	on complete
+		payment 40000
+		dialog
+			`You transfer your sensor logs to Captain Melgrew. She thanks you for your work, and pays you <payment>. She says, "I'll let you know when we have more work."`
+		log "People" "Jane Melgrew" "Jane Melgrew is a Navy captain charged with all manner of odd jobs, from diagnosing hyperspace network problems, to scouting uninhabited systems."
+
+
+
+mission "Republic Scouting 3"
+	name "Navy Reconnaissance"
+	description "Perform an outfit scan on the <npc> in Yed Prior, then report to Captain Melgrew on <destination>."
+	source
+		government "Republic"
+	destination "Follower"
+	to offer
+		has "Republic Scouting 2: done"
+		not "chosen sides"
+		random < 40
+
+	on offer
+		conversation
+			`As you step off your ship, you are greeted by an out-of-breath Jane Melgrew. "Captain <last>!" she exclaims. "Glad I caught you. I have another job, and I couldn't just send you a message for this one.`
+			choice
+				`	"What's going on?"`
+				`	"Sorry, I'm really busy."`
+					decline
+			`	"We have reason to believe there is a ship loaded with alien tech somewhere in the south," she says. "As I'm sure you've heard, the worlds in this area have been organizing a rebellion. So far they've been peaceful, but if they are arming themselves with alien technology, that could be a major problem for us. Would you be able to track down the ship and scan it?"`
+			choice
```

Comment:
I will second Anarchist, this has enormous potential to be a much more fleshed out conversation.